testdata/baselines/reference/submodule/compiler/augmentExportEquals5.types.diff

```diff
@@ -14,6 +14,20 @@
  
          interface IRoute {
              all(...handler: RequestHandler[]): IRoute;
+->all : (...handler: RequestHandler[]) => IRoute
+->handler : RequestHandler[]
++>all : (...handler: (RequestHandler)[]) => IRoute
```

Comment:
Hm, are we missing unsetting something temporarily such that these extra parens appear?